runtime/descriptor.go

```diff
@@ -98,11 +98,13 @@ func propertySet(f *Frame, desc, inst, value *Object) *BaseException {
 
 // makeStructFieldDescriptor creates a descriptor with a getter that returns
 // the field given by fieldName from t's basis structure.
-func makeStructFieldDescriptor(t *Type, fieldName, propertyName string) *Object {
+func makeStructFieldDescriptor(t *Type, fieldName, propertyName string, writtable bool) *Object {
```

Comment:
I would suggest a new `type fieldDescriptorType int` along with fieldDescriptorRO and fieldDescriptorRW enums rather than a writeable bool. It makes calling code less mysterious.